bam_masterdata/datamodel/object_types.py

```diff
@@ -12694,6 +12694,104 @@ class Crystal(MatSimStructure):
     )
 
 
+class MouseMeasurement(ExperimentalStep):
+    defs = ObjectTypeDef(
+        code="EXPERIMENTAL_STEP.MOUSE_MEASUREMENT",
+        description="""Metadata of SAXS measurements of sample at MOUSE // Metadaten der SAXS-Messungen einer Probe mit MOUSE""",
+        generated_code_prefix="EXP.MM_",
+    )
+
+    measurement_date = PropertyTypeAssignment(
+        code="MEASUREMENT_DATE",
+        data_type="DATE",
+        property_label="Measurement Date",
+        description="""Measurement Date//Messdatum""",
+        mandatory=True,
+        show_in_edit_views=False,
+        section="Experiment Details",
+    )
+
+    cell_temperature_in_celsius = PropertyTypeAssignment(
+        code="CELL_TEMPERATURE_IN_CELSIUS",
+        data_type="REAL",
+        property_label="Cell Temperature [°C]",
+        description="""Measurement cell temperature in °C // Temperatur der Messzelle in °C""",
+        mandatory=False,
+        show_in_edit_views=False,
+        section="Experiment Details",
+    )
+
+    exposure_time_in_seconds = PropertyTypeAssignment(
+        code="EXPOSURE_TIME_IN_SECONDS",
+        data_type="REAL",
+        property_label="Exposure time [s]",
+        description="""Exposure time in seconds//Belichtungszeit in Sekunden""",
+        mandatory=True,
+        show_in_edit_views=False,
+        section="Experiment Details",
+    )
+
+    responsible_person = PropertyTypeAssignment(
```

Comment:
🤔 can you elaborate about this property? 

Are you thinking on a situation in which you need to transfer some data from somewhere else (in this case, MOUSE), and then you want to assign the person generating such data in this property?

Is that a "Responsible Person"?